test/inhfd/memfd.py

```diff
@@ -0,0 +1,28 @@
+import os
+import ctypes
+libc = ctypes.CDLL(None)
+
+
+def memfd_create(name, flags):
+    return libc.memfd_create(name.encode('utf8'), flags)
+
+
+def create_fds():
```

Comment:
why this test fails with this patch:
```
diff --git a/test/inhfd/memfd.py b/test/inhfd/memfd.py
index cbf6a0fb2..ddb215b7d 100755
--- a/test/inhfd/memfd.py
+++ b/test/inhfd/memfd.py
@@ -1,4 +1,5 @@
 import os
+import random
 import ctypes
 import subprocess
 libc = ctypes.CDLL(None)
@@ -25,10 +26,11 @@ SYS_memfd_create = __get_SYS_memfd_create()
 
 
 def memfd_create(name, flags):
-    return libc.syscall(SYS_memfd_create, name.encode('utf8'), flags)
+    return libc.memfd_create(name.encode('utf8'), flags)
 
 
 def create_fds():
+    idx = random.randint(0, 1024)
     def create_memfd_pair(name):
         fd = memfd_create(name, 0)
         fw = open('/proc/self/fd/{}'.format(fd), 'wb')
@@ -36,7 +38,7 @@ def create_fds():
         os.close(fd)
         return (fw, fr)
 
-    return [create_memfd_pair("name{}".format(i)) for i in range(10)]
+    return [create_memfd_pair("name{}".format(i + idx)) for i in range(10)]
 
 
 def filename(f):
```

Reply:
It works on my laptop, I'll try on travis-ci

Reply:
I added the added random int commit to another branch: https://github.com/nviennot/criu/commit/72543e4e76b24fb8adbe823b1adb7e230c9dbcc1

So far, the tests are passing: https://travis-ci.org/nviennot/criu/builds/646740596

Reply:
It was my fault. I specified the --iter 10, but with this option, other tests fail too.

Reply:
In travis, the centos job fails. I think you need to add memfd.py.checkskip script:
```
$ cat test/inhfd/memfd.py.checkskip 
#!/usr/bin/env python


import ctypes
libc = ctypes.CDLL(None)

libc.memfd_create("test", 0)
```

Reply:
I updated the branch. There's a failed podman travis test, but that's unrelated.
Otherwise, all the tests are green. Only centos skips the inhfd memfd test, other run it.